platform/windows/godot.natvis

```diff
@@ -138,21 +344,249 @@
 		</Expand>
 	</Type>
 
-	<Type Name="Color">
-		<DisplayString>Color {{{r},{g},{b},{a}}}</DisplayString>
+	<Type Name="godot::Color">
+		<DisplayString>{{{components[0]},{components[1]},{components[2]},{components[3]}}}</DisplayString>
 		<Expand>
-			<Item Name="red">r</Item>
-			<Item Name="green">g</Item>
-			<Item Name="blue">b</Item>
-			<Item Name="alpha">a</Item>
+			<Item Name="red">components[0]</Item>
+			<Item Name="green">components[1]</Item>
+			<Item Name="blue">components[2]</Item>
+			<Item Name="alpha">components[3]</Item>
 		</Expand>
 	</Type>
 
-	<Type Name="Node" Inheritable="false">
+	<Type Name="godot::AABB">
+		<DisplayString>{{{position},{size}}}</DisplayString>
 		<Expand>
-			<Item Name="Object">(Object*)this</Item>
-			<Item Name="class_name">(StringName*)(((char*)this) + sizeof(Object))</Item>
-			<Item Name="data">(Node::Data*)(((char*)this) + sizeof(Object) + sizeof(StringName))</Item>
+			<Item Name="position">position</Item>
+			<Item Name="size">size</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="godot::Transform2D">
+		<DisplayString>{{{elements[0]},{elements[1]},{elements[2]}}}</DisplayString>
+		<Expand>
+			<Item Name="element0">elements[0]</Item>
+			<Item Name="element1">elements[1]</Item>
+			<Item Name="element2">elements[2]</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="godot::Basis">
+		<DisplayString>{{{elements[0]},{elements[1]},{elements[2]}}}</DisplayString>
+		<Expand>
+			<Item Name="x">elements[0]</Item>
+			<Item Name="y">elements[1]</Item>
+			<Item Name="z">elements[2]</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="godot::Transform">
+		<DisplayString>{{{basis},{origin}}}</DisplayString>
+		<Expand>
+			<Item Name="basis">basis</Item>
+			<Item Name="origin">origin</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="godot::String">
+		<DisplayString Condition="((String*)&amp;_godot_string)-&gt;_cowdata._ptr == 0">[empty]</DisplayString>
+		<DisplayString Condition="((String*)&amp;_godot_string)-&gt;_cowdata._ptr != 0">{((String*)&amp;_godot_string)-&gt;_cowdata._ptr,s32}</DisplayString>
+		<StringView Condition="((String*)&amp;_godot_string)-&gt;_cowdata._ptr != 0">((String*)&amp;_godot_string)-&gt;_cowdata._ptr,s32</StringView>
```

Comment:
Note for cherry-picking: In 3.x it should be `,su` (`wchar_t`) instead of `,s32` (`char32_t`).